### PR TITLE
changed to use ur_moder_driver

### PR DESCRIPTION
**Pull request by jrgnicho**
These changes make it possible to run the ur5 robot with a functional version of the ur ros driver that is compatible with the most recent firmwares. @Levi-Armstrong please review.

### CODE REVIEW

**Comment by Levi-Armstrong** [comment]
Do you need this merged for the training class?

File: exercises/Perception-Driven_Manipulation/solution_ws/src/.rosinstall [hunk 1]
```diff
@@ -1,5 +1,7 @@
 - git: {local-name: ar_track_alvar, uri: 'https://github.com/sniekum/ar_track_alvar.git', version: kinetic-devel}
-- git: {local-name: ar_track_alvar_msgs, uri: 'https://github.com/sniekum/ar_track_alvar_msgs.git', version: indigo-devel}
 - git: {local-name: household_objects_database_msgs, uri: 'https://github.com/ros-interactive-manipulation/household_objects_database_msgs.git', version: hydro-devel}
 - git: {local-name: manipulation_msgs, uri: 'https://github.com/ros-interactive-manipulation/manipulation_msgs.git', version: hydro-devel}
 - git: {local-name: universal_robot, uri: 'https://github.com/ros-industrial/universal_robot.git', version: kinetic-devel}
+- git: {local-name: ur_modern_driver, uri: 'https://github.com/geoffreychiou/ur_modern_driver.git', version: iron-kinetic-devel}
+- git: {local-name: ros_control, uri: 'https://github.com/ros-controls/ros_control.git', version: kinetic-devel}
+- git: {local-name: realtime_tools, uri: 'https://github.com/ros-controls/realtime_tools.git', version: kinetic-devel}
```
Comment (gavanderhoorn) [inline exercises/Perception-Driven_Manipulation/solution_ws/src/.rosinstall]: Is there a specific reason why you'd want `ros_control` and `realtime_tools` built from sources?

If you're on Kinetic I'd just use the released versions.

Comment (jrgnicho) [reply exercises/Perception-Driven_Manipulation/solution_ws/src/.rosinstall]: @gavanderhoorn  the ```ros_control``` and ```realtime_tools``` are dependencies of that custom version of ``ur_modern_driver```. That branch is the only one I know works with the latest ur firmware.

Comment (gavanderhoorn) [reply exercises/Perception-Driven_Manipulation/solution_ws/src/.rosinstall]: That may be, but `ros_control` and `realtime_tools` are released packages, for which binary pkgs can be installed using `rosdep install ..`. I was merely curious as to why you included their *source repositories* in the `rosinstall` file.

Comment (jrgnicho) [reply exercises/Perception-Driven_Manipulation/solution_ws/src/.rosinstall]: I see what you mean now,  I copied them from another demo that listed them and didn't spend much time finding out whether the source code was really needed.  Running rosdep install  as you suggested should get the binaries.  I'll do a quick test and submit a PR for remove those repos.

**Pull request merged**